cmd/agent/api/agent/python_status.go

```diff
@@ -7,15 +7,26 @@
 // This group of endpoints is meant to provide high-level functionalities
 // at the agent level.
 
+// +build python
```

Comment:
Should we disable it on Windows since it was crashing there?

```
// +build python !windows
```

Reply:
I intended to test on windows :) Let me double-check that before we remove windows builds :)